packages/api-client/src/api/getProducts/index.ts

```diff
@@ -5,31 +5,31 @@ import { deserializeSearchMetadata } from '../serializers/search';
 export default async function getProducts({ client, config }: ApiContext, params: GetProductsParams): Promise<ProductSearchResult> {
   try {
     const currency = await config.internationalization.getCurrency();
-    const { categoryId, term, optionTypeFilters, productPropertyFilters, priceFilter, page, itemsPerPage, sort } = params;
+    const { categoryId, vendorId, term, optionTypeFilters, productPropertyFilters, priceFilter, page, itemsPerPage, sort } = params;
     let include;
 
     if (config.spreeFeatures.fetchPrimaryVariant) {
-      include = 'primary_variant,default_variant,variants.option_values,option_types,taxons,images';
+      include = 'primary_variant,default_variant,variants.option_values,option_types,taxons,vendor,images';
```

Comment:
This will need to be rewritten and dependent on Spree's config. Vendor is not present in Spree, so this will fail on all Spree instances.

Maybe we should build this `include` param, and just add `vendor` when `config.spreeFeatures.vendoMarketplace` is enabled?